ui/client/components/graph/Graph.js

```diff
@@ -214,12 +214,26 @@ class Graph extends React.Component {
       const source = JointJsGraphUtils.findCell(this.graph, linkBelowCell.attributes.source.id)
       const target = JointJsGraphUtils.findCell(this.graph, linkBelowCell.attributes.target.id)
       const middleMan = cellView.model
+
+      // TODO: handle inject when group is middleman
+      if (NodeUtils.nodeIsGroup(middleMan.attributes.nodeData))  {
```

Comment:
Maybe we should notify user that injecting a group is not possible yet?

Reply:
But show notifiaction type = info, not error :)

Reply:
done